config/dev/cards-publication-dev.yml

```diff
@@ -1,33 +1,4 @@
-server:
-  port: 2102
-spring:
-  application:
-    name: cards-publication
-  deserializer:
-    value:
-      delegate:
-        class: io.confluent.kafka.serializers.KafkaAvroDeserializer
-#        class: org.opfab.cards.publication.kafka.consumer.KafkaAvroWithoutRegistryDeserializer
-  serializer:
-    value:
-      delegate:
-        class: io.confluent.kafka.serializers.KafkaAvroSerializer
-#        class: org.opfab.cards.publication.kafka.producer.KafkaAvroWithoutRegistrySerializer
-# uncomment kafka.consumer.group-id to enable Kafka
-#  kafka:
-#    consumer:
-#      group-id: opfab-command
-
-opfab:
-  kafka:
-    topics:
-      card:
-        topicname: opfab
-      response-card:
-        topicname: opfab-response
-    schema:
-      registry:
-        url: http://localhost:8081
+# This file contains only parameters that needs to be override compare to docker configuration
```

Comment:
that needs => that need
compare to docker configuration => from the docker configuration